compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -901,4 +901,31 @@ object messages {
     val msg = hl"trying to define package with same name as `$existing`"
     val explanation = ""
   }
+
+  case class DangelingThisInPath()(implicit ctx: Context) extends Message(34) {
```

Comment:
Indeed we're up to 35, so right now you'll need a 36. https://github.com/lampepfl/dotty/blob/bf9888edcd96f40f11260e6ad874eb27382f77ba/compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala#L926